style-guide/styles/wds-components/_global-navigation.scss

```diff
@@ -395,4 +372,46 @@
 		}
 	}
 
+	.wds-avatar {
+		@include wds-avatar(30px, $wds-fandom-color-aqua, $wds-color-white, $wds-fandom-color-navy);
+	}
+
+	.wds-button {
+		@include button-theming($wds-fandom-color-aqua, $wds-color-white);
+	}
+
+	.wds-dropdown {
+		height: 100%;
+
+		&__toggle {
+			align-items: center;
+			display: flex;
+			// Safari doesn't set default flex values if we use `flex: 1;`
+			// it requires us to put these values explicitly.
+			flex: 1 1 auto;
+			height: 100%;
+			justify-content: center;
+		}
+
+		&__toggle-chevron {
+			fill: $wds-fandom-color-coral;
+		}
+
+		&__content {
+			border-top: 1px solid $wds-color-light-c4;
+			color: $wds-color-dark-blue-gray;
+			min-width: 190px;
+		}
+		.wds-global-navigation__start-a-wiki {
+			margin: 6px auto;
+		}
+	}
+
+	&__start-a-wiki {
+		&-button:hover {
+			border-color: $wds-color-white!important;
```

Comment:
:see_no_evil:

Reply:
Can you at least make a space between ~$wds-color-white and this terrible exclamation mark? :P